api/v6/container.go

```diff
@@ -54,7 +54,7 @@ func NewContainer(name string, images imageSorter, currentImage image.Info, tagP
 	var c Container
 
 	// The following machinery attempts to minimise the number of
-	// filters (`O(n)`) and sorts (`O(n log n)`), by memoising and
+	// filters (`O(n)`) and sorts (`O(n log n)`), by memorizing and
```

Comment:
This looks like a typo, but is not! https://en.wikipedia.org/wiki/Memoization (or in en_gb, "memoisation")

Reply:
Got it, good lessons. Thanks!